File: Data Structures & Algorithms/reverse-string/submission-0.cpp
class Solution {
public:
    void reverseString(vector<char>& s) {
        int sLen = s.size();
        int left = 0, right = sLen - 1;
        while(left < right){
            swap(s[left], s[right]);
            left++;
            right--;
        }
    }
};

File: Data Structures & Algorithms/depth-of-binary-tree/submission-0.py
# Definition for a binary tree node.
# class TreeNode:
#     def __init__(self, val=0, left=None, right=None):
#         self.val = val
#         self.left = left
#         self.right = right

class Solution:
    
    def maxDepthHelper(self, root: Optional[TreeNode]) -> int:
        if not root:
            return 0
        depthLeft = self.maxDepthHelper(root.left)
        depthRight = self.maxDepthHelper(root.right)
        return 1 + max(depthLeft, depthRight)
    def maxDepth(self, root: Optional[TreeNode]) -> int:
        return self.maxDepthHelper(root)